tox.ini

```diff
@@ -201,6 +201,15 @@ commands =
     {[testenv:full_integration]commands}
     {[testenv]covreport}
 
+[testenv:etl_fast]
+description = Run all CI checks, but for all years of data.
+skip_install = false
+recreate = true
+commands =
+    pudl_setup --pudl_in {env:PUDL_CACHE} --pudl_out {env:PUDL_OUTPUT}
```

Comment:
Are we still using `PUDL_CACHE` for anything or should this be `PUDL_INPUT`?

Reply:
Are the fast ETL outputs mean to be consumed by other tests that aren't doing ETL stuff via use of `--live_dbs`?  While the ETL integration tests will still generate their own DB to generate coverage?

Reply:
I missed responding to this earlier. This is an artifact of when I was pulling the ETL out into its own job and trying to cache the outputs. I imagine that will eventually be useful once we move most of the computation into that phase, and just pop out a `pudl.sqlite` and a few parquet files for the rest of the pipeline, but for now I'm taking it out.